cif2cell/cif2cell.xml

```diff
@@ -0,0 +1,46 @@
+<tool id="cif2cell" name="Convert .cif file to .cell (cif2cell)" version="0.1.0" python_template_version="3.5">
+    <command detect_errors="exit_code"><![CDATA[
+        pip install cif2cell==2.0.0a3 &&
+        ln -s $cif_structure $cif_structure.name &&
+        cif2cell -f $cif_structure.name -p castep -o out.cell &&
+        sed -i '1,8d' out.cell
+    ]]></command>
+    <inputs>
+        <param type="data" name="cif_structure" label="Structure file to Convert (.cif)" />
+    </inputs>
+    <outputs>
+        <data label="Converted structure file of $cif_structure.name (.cell)" name="out_cell" from_work_dir="out.cell" />
+    </outputs>
+    <tests>
+        <test>
+            <param name="cif_structure" value="Si.cif" ftype="txt" />
+            <output name="out_cell" file="Si_out.cell" ftype="data" />
+        </test>
+        <test>
+            <param name="cif_structure" value="diamond.cif" ftype="txt" />
+            <output name="out_cell" file="diamond_out.cell" ftype="data" />
+        </test>
+    </tests>
+    <help><![CDATA[
+        usage: cif2cell -f file.cif -p castep -o file.cell
+
+        Given an input .cif structure file, convert to equivalent .cell structure file.
+    ]]></help>
+    <citations>
+        <citation type="bibtex">
```

Comment:
Not sure why, but this citation isn't appearing when I load the tool. Is it formatted correctly?

![image](https://user-images.githubusercontent.com/20683271/160586017-d798d931-96fa-4b82-b90e-eecda80f499e.png)